modules/table/src/main/java/org/apache/ignite/internal/table/distributed/replicator/PartitionReplicaListener.java

```diff
@@ -468,6 +481,10 @@ public CompletableFuture<ReplicaResult> invoke(ReplicaRequest request, String se
                 });
     }
 
+    private CompletableFuture<Void> switchToRequestOperationsThread() {
+        return runAsync(NO_OP, requestOperationsExecutor);
```

Comment:
Wondering whether it's an enough efficient why to switch the thread. Any thoughts?